core/Cargo.toml

```diff
@@ -39,22 +39,23 @@ flate2 = { version = "1.0", optional = true }
 futures = "0.3"
 futures-util = "0.3"
 governor = "0.6"
-http = "0.2"
-hyper = "0.14"
+http-body-util = { version = "0.1", optional = true }
+hyper = { version = "1.2", optional = true }
```

Comment:
Note that tonic 0.11 still depends on hyper 0.14, so this duplicates a bunch of the hyper stack in the dependencies.